rmgpy/reactionTest.py

```diff
@@ -46,7 +46,9 @@
 from rmgpy.statmech.vibration import Vibration, HarmonicOscillator
 from rmgpy.statmech.torsion import Torsion, HinderedRotor
 from rmgpy.statmech.conformer import Conformer
-from rmgpy.kinetics import Arrhenius, ArrheniusEP, SurfaceArrhenius, StickingCoefficient
+from rmgpy.kinetics import Arrhenius, ArrheniusEP, MultiArrhenius, \
```

Comment:
Can you also remove some of the unused imports in this file? For example, from `rmgpy.quantity import Quantity` is stated twice.

Reply:
Done